pkgs/development/python-modules/pandas-stubs/default.nix

```diff
@@ -1,39 +1,117 @@
 { lib
 , buildPythonPackage
-, isPy27
 , fetchFromGitHub
+, jinja2
+, matplotlib
+, openpyxl
+, pandas
+, poetry-core
+, scipy
+, sqlalchemy
+, tabulate
+, pyarrow
+#, pyreadstat
+, tables
+, pytestCheckHook
+, pythonOlder
+, types-pytz
 , typing-extensions
-, mypy
+, xarray
 }:
 
 buildPythonPackage rec {
   pname = "pandas-stubs";
-  version = "1.2.0.39";
+  version = "1.5.0.221003";
+  format = "pyproject";
 
-  disabled = isPy27;
+  disabled = pythonOlder "3.8";
 
-  # Use GitHub source since PyPi source does not include tests
   src = fetchFromGitHub {
-    owner = "VirtusLab";
+    owner = "pandas-dev";
     repo = pname;
-    rev = "2bd932777d1050ea8f86c527266a4cd205aa15b1";
-    sha256 = "m2McU53NNvRwnWKN9GL8dW1eCGKbTi0471szRQwZu1Q=";
+    rev = "v${version}";
+    sha256 = "sha256-RV0pOTPtlwBmYs3nu8+lNwVpl/VC/VzcXKOQMg9C3qk=";
   };
 
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
   propagatedBuildInputs = [
+    pandas
+    types-pytz
+  ];
+
+  checkInputs = [
+    jinja2
+    matplotlib
+    openpyxl
+    scipy
+    sqlalchemy
+    tabulate
+    pyarrow
+    tables
+#    pyreadstat
```

Comment:
There is an open PR for that IIRC

Reply:
Indeed, https://github.com/NixOS/nixpkgs/pull/192277